src/pint/models/binary_ell1.py

```diff
@@ -117,7 +117,7 @@ def __init__(self):
             floatParameter(
                 name="EPS1",
                 units="",
-                description="First Laplace-Lagrange parameter, ECC x sin(OM) for ELL1 model",
+                description="First Laplace-Lagrange parameter",
```

Comment:
Why remove the specificity? Isn't it helpful?

Reply:
The long descriptions were causing the summary table to be too wide.

Reply:
added the expression again.